app/components/file-browser/component.ts

```diff
@@ -156,6 +155,20 @@ export default class FileBrowser extends Component.extend({
         return guid ? pathJoin(window.location.origin, guid) : undefined;
     }
 
+    @computed('canEdit', 'loading', 'uploading', 'items', 'filter')
+    get clickable(this: FileBrowser) {
+        const cssClass = ['.dz-upload-button'];
```

Comment:
I'm really not a fan of passing a list of classes like this, and at some point we should rethink how the dropzone widget works, but I think it's reasonable to put that outside the scope of this bugfix.

Reply:
Pretty sure that's just how dropzone.js accepts clickable elements: selectors or elements, or an array of either.

Reply:
Yeah, but that seems like something that should be hidden inside the `dropzone-widget` component. But maybe that's unreasonable, I dunno.